numpy/tests/test_public_api.py

```diff
@@ -246,18 +246,24 @@ def test_NPY_NO_EXPORT():
     "distutils.unixccompiler",
     "dual",
     "f2py.auxfuncs",
+    "f2py.backends",
```

Comment:
This does not look like it should be public, or should it? If not, can you please name it `f2py._backends`?

Reply:
And the same for all the other things - nothing should be added to this list ideally.